.github/workflows/codeflare-sdk-release-sync.yml

```
@@ -0,0 +1,92 @@
+---
+# The aim of this GitHub workflow is to update the pipfile to sync with Codeflare-SDK release.
+name: Sync with codeflare-sdk release
+on:  # yamllint disable-line rule:truthy
+  workflow_dispatch:
+    inputs:
+      codeflare_sdk_release_version:
+        required: true
+        description: "Provide the version of the codeflare release you want to update to : "
+      branch:
+        required: true
+        description: "Provide the name of the branch you want to update ex main, vYYYYx etc: "
+
+env:
+  BRANCH_NAME: ${{ github.event.inputs.branch || 'main' }}
+  CODEFLARE_RELEASE_VERSION: ${{ github.event.inputs.codeflare_sdk_release_version }}
+  UPDATER_BRANCH: codeflare-sync-updater-${{ github.run_id }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

```suggestion
      - uses: actions/checkout@v4
```
To address deprecation warning.